GooglePlayInstant/LoadingScreen/LoadingBar.cs

```diff
@@ -41,19 +41,18 @@ public static class LoadingBar
 
         private const int LoadingBarHeight = 30;
 
-        // Loading bar width as a percentage canvas object's automatic size
-        private const float LoadingBarWidthPercentage = .5f;
+        // Loading bar width as a percentage of the screen width.
+        private const float LoadingBarWidthPercentage = .7f;
```

Comment:
Nit: note it's not a percentage since that'd be 70 instead of 0.7
```
        // Loading bar size as a proportion of the screen size. Adjust if needed.
        private const float LoadingBarWidthProportion = 0.7f;
        private const float LoadingBarHeightProportion = 0.1f;
```
Not sure about that last number; adjust as needed.